README.md

```diff
@@ -18,427 +18,15 @@ A modern, lightweight learning management system.
 
 ## Getting started
 
-### Getting started on OS X
+### Getting started on MacOS, Linux or Docker.
```

Comment:
No need for this header in fact... we only have one `<h3>` it seems in this section.